tools/cmake/Makefile

```diff
@@ -7,13 +7,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cmake
-PKG_VERSION:=3.12.4
+PKG_VERSION:=3.13.2
 PKG_CPE_ID:=cpe:/a:kitware:cmake
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://cmake.org/files/v3.12/ \
```

Comment:
https://github.com/Kitware/CMake seems to be a mirror of https://gitlab.kitware.com/cmake/cmake; wouldn't it be better to use that one ?

Reply:
I just used what the official site links to:
https://cmake.org/download/ --> https://github.com/Kitware/CMake/releases/download/v3.13.2/cmake-3.13.2.tar.gz

Reply:
@diizzyy Checked https://cmake.org/download/; I don't find any clear statement main URL changed to GitHub unless I'm missing something. All I can find is a reference to GitHub for older releases

Reply:
Hover over the download link? :-)

Reply:
@diizzyy I'm thinking they want to keep https://cmake.org/download as the 'official' link in case they change release / git hosting.

Reply:
It's a static file and it's what upstream "suggests"
![cmake](https://user-images.githubusercontent.com/11659781/50528038-4d7b5a00-0aec-11e9-812d-4a088be88164.jpg)

Reply:
@diizzyy it's easy enough to update a webpage / website without anyone noticing except the maintainer.  I did see that's where they link, but the point is their intent may be to keep control of the download link in case something changes with their wish to use github without users continuing to use github not realizing it is no longer the 'official' project.

Reply:
My preference is to keep cmake.org/files/v3.13 as PKG_SOURCE_URL; I want to avoid frequent switches in the future unless a clear statement is made the official download link is changed.
I will close this PR as I've https://git.openwrt.org/?p=openwrt/staging/dedeckeh.git;a=commit;h=d9bbd02920160836da9f6a40d64c2ffb7672c2bc in my staging tree